src/Pass/ImgTagTransformPass.php

```diff
@@ -110,6 +201,19 @@ protected function getImageWidthHeight($src)
 
         // Try obtaining image size without having to download the whole image
         $size = $this->fastimage->getImageSize($img_url);
```

Comment:
@mpatnode-si @FuzzyTree is planning to try to contribute back his changes to the main repo so that we don't have to maintain a fork. While there may be a performance improvement by checking for our specific image pathnames, this would not be something that we could get merged into the main repo. Seems like this is fine since people getting our content from Google's CDN and not from our site directly. What do you think?